docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -10,6 +10,39 @@ A replication factor of at least 3 is highly recommended for any M3DB deployment
 
 M3DB will do its best to distribute shards evenly among the availability zones while still taking each individual node's weight into account, but if some of the availability zones have less available hosts than others then each host in that zone will be responsible for more shards than hosts in the other zones and will thus be subjected to heavier load.
 
+
+### Replication Factor Recommendations
+
+Running with `RF=1` or `RF=2` is not recommended for any multi-node use cases (testing or production). In [the
+future][1107] such topologies may be rejected by M3DB entirely. It is also recommended to only run with an odd number of
```

Comment:
This seems aggressive. Its still really useful for running docker containers on limited hardware (like a laptop) to only have one node

Reply:
@richardartoul yeah I can try to clarify this more. I said not recommended for "multi-node use cases" and by multi-node testing I meant as in testing a deployment of M3, not as much local testing